### DISCUSSION TITLE
GoogleMaps Support

### DISCUSSION THREAD

**Discussion by kartikhedau** [TDesjardins/gwt-ol3#192] [Q&A] [answered]
Does GWT-OL support Google Maps APIs? From the examples, I could only find support for Bing Maps and OpenStreetMap.

**Comment by TDesjardins** [comment, marked as answer]
Hello [kartikhedau](https://github.com/kartikhedau),

it is a bit late answer. GWT-OL doesn't support Google Maps API because OpenLayers doesn't support it. As far as I know the simple reason is that the terms of Google Maps don't allow it.
